lib/test.js

```diff
@@ -1348,6 +1348,56 @@ class Test extends Base {
         t.match(value, wanted, extra)
       }))
   }
+
+  emitsEvent (emitter, event_name, message, extra) {
```

Comment:
camels, please, snake_case is too scary because snakes.

Reply:
will do